main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -84,6 +85,7 @@ class Flix {
     "Result.flix" -> LocalResource.get("/library/Result.flix"),
     "Set.flix" -> LocalResource.get("/library/Set.flix"),
     "String.flix" -> LocalResource.get("/library/String.flix"),
+    "Zip.flix" -> LocalResource.get("/library/Core/Io/Zip.flix"),
```

Comment:
Group these together below so the structure becomes more easy to see.
Rename `InputStream.flix -> Core/Io/InputStream.flix` same for Flix. Otherwise the paths are not printed correctly.